packages/xo-web/src/common/xo/index.js

```diff
@@ -1010,14 +1023,32 @@ export const startVms = vms =>
       )
     )
 
-    if (forbiddenStart.length === 0) {
+    if (forbiddenStart.length === 0 && vmsWithduplicatedMacAddresses.length === 0) {
       if (nErrors === 0) {
         return
       }
 
       return error(_('failedVmsErrorTitle'), _('failedVmsErrorMessage', { nVms: nErrors }))
     }
 
+    if (vmsWithduplicatedMacAddresses.length > 0) {
+      // Retry without checking MAC addresses
+      try {
+        await confirm({
+          title: _('forceStartVm'),
+          body: _('vmsWithDuplicatedMacAddressesMessage', { nVms: vmsWithduplicatedMacAddresses.length }),
+        })
+        await Promise.all(map(vmsWithduplicatedMacAddresses, id => _startVm(id, undefined, true, true)))
+      } catch (error) {
+        // Nothing to do
```

Comment:
What if some of the VMs that failed because of duplicated MAC addresses also have `start` in their blocked operations? https://github.com/vatesfr/xen-orchestra/pull/5655#discussion_r596658735

Reply:
We will force the start. Does that answer your question?
https://github.com/vatesfr/xen-orchestra/blob/cf320c08c5204043b70c5d626c8ac34d25b24e23/packages/xo-server/src/xapi/index.js#L1366-L1381